packages/node-core/src/indexer/storeCache/cacheModel.ts

```diff
@@ -234,7 +236,7 @@ export class CachedModel<
   }
 
   clear(): void {
-    this.getCache = {};
+    this.getCache.clear();
```

Comment:
Is this something we still want? We want the cache to persist across blocks right? I think clear would get called every time we flush